faqs/galaxy/purge histories

```diff
@@ -0,0 +1,15 @@
+---
+title: Find all Histories and purge (aka permanently delete)
+area: Account      
```

Comment:
I don't know about "area: Account". Could it be more "histories"?

Reply:
thank you , updated to Histories